autofit/graphical/mean_field.py

```diff
@@ -434,19 +440,23 @@ def project_mean_field(
     ) -> Tuple["FactorApproximation", Status]:
         success, messages = Status() if status is None else status
 
-        factor_dist = (model_dist / self.cavity_dist)
-        if delta < 1:
-            log_norm = factor_dist.log_norm
-            factor_dist = (
-                    factor_dist ** delta * self.factor_dist ** (1 - delta))
-            factor_dist.log_norm = (
-                    delta * log_norm + (1 - delta) * self.factor_dist.log_norm)
-
-        if not factor_dist.is_valid:
-            success = False
-            messages += (
-                f"model projection for {self} is invalid",)
-            factor_dist = factor_dist.update_invalid(self.factor_dist)
+        try:
```

Comment:
@matthewghgriffiths not sure if this is the best idea but it was necessary for easily testing other functionality using mock objects